Sources/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -455,7 +455,7 @@ extension NSManagedObjectContext: MessageDatabaseSession {
             throw ClientError.ChannelDoesNotExist(cid: cid)
         }
 
-        let message = MessageDTO.loadOrCreate(id: .newUniqueId, context: self, cache: nil)
+        let message = MessageDTO.loadOrCreate(id: .newUniqueId.lowercased(), context: self, cache: nil)
```

Comment:
Hi @Aurazion ! Thanks for the PR! Can you actually put the `lowercased()` inside the `newUniqueId`? So that it is consistent across all our UUID's generations? Thanks!

Also, can you specify which issues are you having?

Reply:
Hi @nuno-vieira ! I applied your changes thanks 👍

We having issues when we tried to request some messages on your API (case sensitive visibly) because we store all UUID in lowercase in our backend for later use. And we saw that every other platforms generate it by default in lowercase. So when we try from our backend to query some messages sent from iOS, your API is unable to fetch the message because of the case sensitivity.